crates/wasmtime/src/runtime/component/mod.rs

```diff
@@ -672,3 +672,158 @@ pub mod bindgen_examples;
 #[cfg(not(any(docsrs, test, doctest)))]
 #[doc(hidden)]
 pub mod bindgen_examples {}
+
+#[cfg(not(feature = "component-model-async"))]
+pub(crate) mod concurrent {
```

Comment:
To confirm, this'll eventually be a `concurrent.rs` in a future PR? So just a placeholder for now?

Reply:
`concurrent.rs` is [already in main](https://github.com/bytecodealliance/wasmtime/blob/main/crates/wasmtime/src/runtime/component/concurrent.rs) and was added in https://github.com/bytecodealliance/wasmtime/pull/10044 (and will be expanded in subsequent PRs).  This is the stub module to be used when the `component-model-async` feature is disabled, so it will only go away when that feature is removed.

Alternatively, I could remove this stub module and make `concurrent.rs` the only implementation, but that would mean adding more `#[cfg(...)]` annotations elsewhere, so it's kind of a tradeoff.  Happy to do whichever you think is best, though.

Reply:
Ah I see, sorry I missed the `not(...)`!

I'll take a closer look in a future PR to see how it fits together 👍